test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs

```diff
@@ -745,6 +746,123 @@ public async Task AddPkg_ConditionalAddWithUserInputFramework_Success(string pac
             }
         }
 
+        [Fact]
+        public async Task AddPkg_WithAdditionalSourceFeed_NoVersionSpecified_Success()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var projectFrameworks = "net472";
+                var packageFrameworks = "net472; netcoreapp2.0";
+                var projectA = XPlatTestUtils.CreateProject(ProjectName, pathContext, projectFrameworks);
+                var packageX = XPlatTestUtils.CreatePackage(frameworkString: packageFrameworks);
+                var packageXPath = Path.Combine(pathContext.WorkingDirectory, "packageX");
+                var mySourcePath = Path.Combine(pathContext.WorkingDirectory, "MySource");
+
+                Directory.CreateDirectory(mySourcePath);
+
+                // Generate Package
+                await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    packageXPath,
+                    PackageSaveMode.Defaultv3,
+                    packageX);
+
+                string originalPackageXNupkg = Path.Combine(packageXPath, packageX.Id, packageX.Version, $"{packageX.Id}.{ packageX.Version}.nupkg");
+                string mySourcePackageXNupkg = Path.Combine(mySourcePath, $"{packageX.Id}.{ packageX.Version}.nupkg");
+
+                if (!File.Exists(originalPackageXNupkg))
+                {
+                    // On linux it's created in different place.
```

Comment:
why is it created in a different place?

Reply:
I don't know for now because I don't have any machine with Linux.
But I did create follow up [issue#10361](https://github.com/NuGet/Home/issues/10361)